modules/oadp-installing-oadp-rosa-sts.adoc

```diff
@@ -34,6 +34,13 @@ web_identity_token_file = /var/run/secrets/openshift/serviceaccount/token
 EOF
 ----
 
+.. Create a namespace for OADP:
++
+[source, terminal]
```

Comment:
This seems to be fine in the preview build, but I am not sure the portal transform will like it.
```suggestion
[source,terminal]
```